### DISCUSSION TITLE
Need to add a video current position slider

### DISCUSSION THREAD

**Discussion by TwoPony** [IonDen/ion.rangeSlider#765] [Ideas]
I have a video playing with a range slider allowing the user to constrain the starting and ending points to be viewed.  No problem.
What I want to add is an indicator on that same range slider that shows where the video is right now - this would help the viewer when she wants to limit the playback to a certain range in the clip. - Slide the starting slider to close to where the indicator is, and then do the same when the video is about at the desired ending position.

**Comment by IonDen** [comment]
Hi, sorry but I didn't understood your question.
